gatsby/content/projects/clients/quadrix.mdx

```diff
@@ -0,0 +1,66 @@
+---
+layout: projectimage
+title: quadrix.chat
+slug: quadrix
+categories:
+ - client
+description: Minimal Matrix client available in all main app stores
+author: Jean-François Alarie
+home: https://quadrix.chat
+language: Typescript
+license: GPL-3.0-or-later
+repo: https://github.com/alariej/quadrix
+room: "#quadrix:matrix.org"
+thumbnail: /docs/projects/images/quadrix-screenshot.png
+screenshot: /docs/projects/images/quadrix-screenshot.png
+maturity: Released
+platforms:
+    - Linux
+    - Mac
+    - Windows
+    - Web
+    - Android
+    - iOS
+    - Mobian
+featured: true
```

Comment:
Tested locally and the project page looks good:

http://localhost:8000/docs/projects/client/quadrix
<img width="1189" alt="" src="https://user-images.githubusercontent.com/558581/183513631-0c617a8e-092b-45a2-b3b6-5fbb4ee26993.png">